libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1159,26 +1159,28 @@ void IRGeneratorForStatements::appendExternalFunctionCall(
 			if iszero(extcodesize(<address>)) { revert(0, 0) }
 		</checkExistence>
 
-		let <pos> := <freeMem>
+		let <pos> := mload(<freeMemoryPointer>)
```

Comment:
Why not `fetchFreeMemoryPointer` here?

Reply:
We need `freeMemoryPointer` a bit further down so it seemed easier to just use one variable used at both places instead of adding `fetchFreeMemoryPointer` for this one use

Reply:
I think it is worthwhile adding another parameter.